Readme.md

```diff
@@ -192,18 +194,18 @@ There are several properties exposed from the hook to deal with the state throug
 | error            | any error thrown within the handleSubmit function is stored here as an error object                    |
 | submissionStatus | Displays the current status of the submission process                                                  |
 
-#### Submission Status
+### Submission Status
 
 This variable contains the current state of the form submission process.
 
 | Status     | Details                                                                     |
 | ---------- | --------------------------------------------------------------------------- |
-| idle       | The form has not yet been submitted. It also applied when the form is reset |
+| pristine   | The form has not yet been submitted. It also applied when the form is reset |
```

Comment:
For me Idle suggests something happening in the background, and idle means it's not happening anymore. i like Angular's wording here more `Pristine` means untouched

Reply:
I think idle is a synonym of "stand by" it means waiting to do something. The flow is:

idle -> submitting -> submitted / error -> reset -> idle

https://www.dictionary.com/browse/idle